main.py

```diff
@@ -205,6 +216,22 @@ def loadDirectory(self):
                 self._BLOCK_STRUCTURES_ = False
             return 1
 
+    def deleteLoadedFiles(self):
+        #clearing all widgets it's not a problem even if it does not exist
+        # self.deleteWidget(0) #to show msg
+        for i in range(WidgetHandler.visibleCounter):
+            if i == 0:
+                self.deleteWidget(i)
+                continue
+            self.deleteWidget(i, False)
+
+        self.doh = DataObjectHolder()
```

Comment:
ok this is a good idea but can we do smth with doh reinstatiation? It might be sufficient to clean it, because it might contain other structs necessary for the app, not particularly for widget